index.js

```diff
@@ -1,28 +1,34 @@
+/* eslint-disable no-multiple-empty-lines */
+/* eslint-disable no-trailing-spaces */
+/* eslint-disable no-empty */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-multi-spaces */
+/* eslint-disable quotes */
 /* eslint-disable prefer-destructuring */
```

Comment:
- [ ] As the previous reviewer suggested, you should try as much as possible to avoid disabling linter errors in the file. This is to ensure that clean code that is easy to maintain is written. In addition,  linters enable the automatic detection of vulnerabilities and customizable rules ☺️.

Reply:
Thanks, @AlphaNtihinduka This is eslint that I installed on my Vs code from the extensions tap, not the eslint installed as described in the how to install javascript eslint in the project description. Please reply for further action as I am not very sure.

Reply:
Hi @enocol Please setup the linters using the files provided by the microverse. [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js) is the link to it.  There is no need for any VS extension, the files are enough for performing linter checks.